cmd/bom/cmd/generate.go

```diff
@@ -174,7 +174,7 @@ completed by a later stage in your CI/CD pipeline. See the
 	if err := generateCmd.PersistentFlags().MarkDeprecated(
 		"tarball", "tarball has been renamed to image-archive",
 	); err != nil {
-		logrus.Fatal(errors.Wrap(err, "marking flag as deprecated"))
+		logrus.Fatal(fmt.Errorf("marking flag as deprecated: %w", err))
```

Comment:
```suggestion
		logrus.Fatalf("marking flag as deprecated: %v", err)
```

Reply:
Ack. Addressed in the latest comment. Thank you for pointing.